tests/api/BoreholeLockServiceTest.cs

```diff
@@ -106,6 +106,53 @@ public async Task CreateForLockedBoreholeWithElapsedLockTimeout()
         Assert.AreEqual(false, await boreholeLockService.IsBoreholeLockedAsync(borehole.Id, AdminSubjectId));
     }
 
+    [TestMethod]
+    public async Task GetBoreholesUserLacksPermissionWithSufficientPermissions()
+    {
+        var user = await context.Users
+            .Include(s => s.WorkgroupRoles)
+            .SingleOrDefaultAsync(u => u.FirstName == "Editor");
+
+        var boreholes = await context.Boreholes.Include(b => b.Workflows).ToListAsync();
+        boreholes = boreholes.Where(x => x.Workflows.Count > 0 && x.Workflows.Any(w => w.Role == Role.Editor)).Take(2).ToList();
+
+        var result = boreholeLockService.GetBoreholesUserLacksPermissionFor(boreholes, user);
+        Assert.AreEqual(0, result.Count);
+    }
+
+    [TestMethod]
+    public async Task GetBoreholesUserLacksPermissionWithNotMatchingWorkGroupId()
```

Comment:
PP: Workgroup anstatt WorkGroup
```suggestion
    public async Task GetBoreholesUserLacksPermissionWithNotMatchingWorkgroupId()
```